db_test.go

```diff
@@ -387,6 +386,23 @@ func TestGetMore(t *testing.T) {
 			txn.Discard()
 		}
 
+		// MultiGet
+		var multiGetKeys [][]byte
+		var expectedValues []string
+		for i := 0; i < n; i += 100 {
+			multiGetKeys = append(multiGetKeys, data(i))
+			expectedValues = append(expectedValues, fmt.Sprintf("zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz%9d", i))
```

Comment:
Remove "zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz"

Reply:
This test needs the value to be large enough, so there would be multiple levels in DB.

Reply:
please add comments.